### PR TITLE
Redesign Tech List

### PR DESCRIPTION
**Pull request by moralesbang**
## Summary

- Redesigned tech list view

## Screenshots

Screen Shot 2019-07-19 at 5 16 24 PM

## Trello Card

https://trello.com/c/Qm7jQJ1S

### CODE REVIEW

File: maquetado/scss/_tech.scss [hunk 1]
```diff
@@ -52,15 +37,12 @@
...
-  }
-
-  .tech {
-    margin: 20px;
-  }
+  align-items: center;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(3, $tech-width);
+  grid-template-rows: repeat(2, auto);
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: I think that this is unnecessary, please delete it

Comment (moralesbang) [reply maquetado/scss/_tech.scss]: Delete what? Line #44?

Comment (henryzarza) [reply maquetado/scss/_tech.scss]: Yes

File: maquetado/scss/_tech.scss [hunk 2]
```diff
@@ -52,15 +40,10 @@
...
-    margin-bottom: 100px;
-  }
-
-  .tech {
-    margin: 20px;
-  }
+  align-items: center;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(3, $tech-width);
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]:
```suggestion
  grid-template-columns: repeat(3, 1fr);
```
For the responsive 😉

Comment (moralesbang) [reply maquetado/scss/_tech.scss]: Si agrego tu sugerencia, pasa esto:

Screen Shot 2019-07-22 at 9 36 25 AM

Comment (henryzarza) [reply maquetado/scss/_tech.scss]: Yes, my mistake. I'm sorry

**Pull request merged**